test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/TestUtilities.cs

```diff
@@ -10,6 +16,8 @@ namespace Microsoft.AspNet.Authentication.Tests.OpenIdConnect
     /// </summary>
     public class TestUtilities
     {
+        public const string DefaultHost = @"http://localhost";
+
         public static bool AreEqual<T>(object obj1, object obj2, Func<object, object, bool> comparer = null) where T : class
```

Comment:
How is this method different from the existing xUnit.net method? https://github.com/xunit/xunit/blob/master/src/xunit.assert/Asserts/EqualityAsserts.cs#L30